packages/components/src/components/link/component.tsx

```diff
@@ -56,7 +59,9 @@ export class KolLinkWc implements LinkAPI {
 	};
 
 	private readonly onClick = (event: Event) => {
-		if (typeof this.state._on?.onClick === 'function') {
+		if (this.state._disabled === true) {
+			preventDefault(event);
```

Comment:
```suggestion
			preventDefaultAndStopPropagation(event);
```

Kleiner Refactoring-Vorschlag. Ich finden aktuellen Funktionsnamen irreführend.